### PR TITLE
HAAR-116 Publish PRISON_REGISTER_UPDATE event to Topic when prison changed

### PR DESCRIPTION
**Pull request by karenmillermoj**

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/prisonregister/resource/PrisonMaintenanceResourceIntTest.kt [hunk 1]
```diff
@@ -94,6 +109,8 @@ class PrisonMaintenanceResourceIntTest : IntegrationTest() {
         .expectBody().json("updated_prison".loadJson())
 
       verify(auditService).sendAuditEvent("PRISON_REGISTER_UPDATE", Pair("MDI", UpdatePrisonDto("Updated Prison", false)))
+      verify(snsService).sendEvent(EventType.PRISON_REGISTER_UPDATE, "MDI")
+      verify(telemetryClient).trackEvent(eq("prison-register-update"), any(), isNull())
     }
   }
```
Comment (karenmillermoj) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/prisonregister/resource/PrisonMaintenanceResourceIntTest.kt:115]: Next PR will add a test with a test-queue subscribing to the domain events topic, ensuring the event is received by the test queue

File: README.md [hunk 2]
```diff
@@ -31,6 +31,37 @@ SPRING_PROFILES_ACTIVE=postgres,local ./gradlew bootRun
...
 You can obtain valid auth tokens form the local auth server using curl.
+
+### Running in IntelliJ
+To run a local version with a postgres h2 db, simply use the dev spring profile.
+
+#### LocalStack & AWS
+Localstack and AWS use different bean config functions. So for each AWSSQSClient there will be 2 versions of this
+config.
+
+Queues, topics and the tables will automatically when the application starts.
```
Comment (neilmendum) [inline README.md:42]: Think you missed a word here - _automatically be created_?

File: docker-compose.yml [hunk 3]
```diff
@@ -14,7 +14,7 @@ services:
       test: [ "CMD", "curl", "-f", "http://localhost:8080/health/ping" ]
     environment:
       - SERVER_PORT=8080
-      - OAUTH_API_BASE_URL=http://localhost:9090/auth
+      - OAUTH_API_BASE_URL=http://localhost:8080/auth
```
Comment (neilmendum) [inline docker-compose.yml:17]: I think this can stay as `9090` as that is the `hmpps-auth` container's published port

Comment (karenmillermoj) [reply docker-compose.yml:17]: good spot, think I was a bit hasty

**Review by neilmendum — APPROVED** [review]
LGTM

**Pull request merged**